spec/order_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'order'
+
+describe Order do
+
+  subject(:order) { described_class.new }
+  let(:menu) { double :menu, :food_options => { 'Spring rolls' => 2.50 }}
+
+  describe 'customers can place orders' do
+    it 'is expected to repeat customer order' do
+      expect(order.order_received('Spring rolls', 1)).to eq "Order received for 1 Spring rolls."
+    end
+
+    it 'raises an error if the food order does not exist' do
+      expect{subject.order_received('Pizza', 1)}.to raise_error "There is no Pizza available on the menu"
+    end
+  end
+
+  describe 'the bill is calculated throughout and can be checked' do
+    it 'adds cost of items to order bill' do
+      order.order_received('Spring rolls', 12)
+      expect(order.bill).to eq 30
+    end
+
+    it 'can check bill' do
+      order.order_received('Spring rolls', 2)
+      expect(order.check_total).to eq "The bill is currently £5.00."
+    end
+
+    it 'raises an error if bill does not have same value as combined selection price', :focus => true do
+      allow(order).to receive(:bill_correct?).and_return false
+      expect{order.check_total}.to raise_error "There is an error - your bill is not the same as the item cost."
+    end
+  end
+
+  describe 'is saved and can be requested by the customer' do
+
```

Comment:
Try and be consistent with your whitespace usage.  There's an empty line after this describe line, but not after the other describe lines in the same project.